From 2e4e3bc7259d2af97437769091d132c7a9a81324 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:33 -0400
Subject: [usb] kill #undef VERBOSE_DEBUG

Message-id: <1410887463-20674-202-git-send-email-dzickus@redhat.com>
Patchwork-id: 95168
O-Subject: [RHEL7 PATCH 201/591] USB: kill #undef VERBOSE_DEBUG
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit c793d08ecd944b1a180c6712bb47825e37cc0626
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon Nov 18 13:23:17 2013 +0100

    USB: kill #undef VERBOSE_DEBUG

    It is useless now. Straight removal.

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ehci-hcd.c b/drivers/usb/host/ehci-hcd.c
index bfcc140..7c1bf30 100644
--- a/drivers/usb/host/ehci-hcd.c
+++ b/drivers/usb/host/ehci-hcd.c
@@ -71,7 +71,6 @@
 static const char hcd_name [] = "ehci_hcd";
 
 
-#undef VERBOSE_DEBUG
 #undef EHCI_URB_TRACE
 
 /* magic numbers that can affect system performance */
diff --git a/drivers/usb/host/ohci-hcd.c b/drivers/usb/host/ohci-hcd.c
index 40cf076..290174a 100644
--- a/drivers/usb/host/ohci-hcd.c
+++ b/drivers/usb/host/ohci-hcd.c
@@ -51,8 +51,6 @@
 
 /*-------------------------------------------------------------------------*/
 
-#undef OHCI_VERBOSE_DEBUG /* not always helpful */
-
 /* For initializing controller (mask in an HCFS mode too) */
 #define OHCI_CONTROL_INIT OHCI_CTRL_CBSR
 #define OHCI_INTR_INIT \
-- 
1.7.1